src/index.js

```diff
@@ -0,0 +1,108 @@
+
+const apiUrl = 'https://api.tvmaze.com/shows';
+const likesApiUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/EK8AqlUP7MtIYG7gJYqn/likes/';
+const movieCardsContainer = document.getElementById('movie-cards');
+const pop = document.querySelector('.pop');
+
+const fetchMovieData = async (showId) => {
+  const response = await fetch(`${apiUrl}/${showId}`);
+  const data = await response.json();
+  return {
+    name: data.name,
+    image: data.image.medium,
+    genres: data.genres,
+    itemId: showId,
+  };
+};
+
+const fetchLikesData = async (showId) => {
+  const response = await fetch(`${likesApiUrl}?item_id=${showId}`);
+  const data = await response.json();
+  // eslint-disable-next-line camelcase
+  const res = data.find(({ item_id }) => item_id === showId);
+  return res ? res.likes : 0;
+};
+
+const updateLikesData = async (showId, likes) => {
+  const response = await fetch(`${likesApiUrl}`, {
+    method: 'POST',
+    body: JSON.stringify({ item_id: showId, likes }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+  const data = await response.text();
+  return data;
+};
+
+const createMovieCard = async (movieData, showId) => {
+  const card = document.createElement('div');
+  card.classList.add('movie-card');
+  card.id = `movie-${showId}`;
+
+  const title = document.createElement('h2');
+  title.innerText = movieData.name;
+
+  const image = document.createElement('img');
+  image.src = movieData.image;
+
+  const comment = document.createElement('button');
+  comment.innerHTML = 'Comment';
+  comment.classList.add('comment');
+
+  const likeBtn = document.createElement('button');
+  likeBtn.classList.add('likes');
+  likeBtn.innerHTML = 'Like';
+
+  const likes = document.createElement('p');
+  likes.innerHTML = 'Likes: 0';
+
+  const movieLikes = await fetchLikesData(showId);
+  likes.innerHTML = `Likes: ${movieLikes}`;
+
+  likeBtn.addEventListener('click', async () => {
+    const currentLikes = parseInt(likes.innerHTML.split(' ')[1], 10);
+    const newLikes = currentLikes + 1;
+    likes.innerHTML = `Likes: ${newLikes}`;
+    await updateLikesData(showId, newLikes);
+  });
+
+  const genres = document.createElement('p');
+  genres.innerHTML = `<strong>Genres:</strong> ${movieData.genres.join(', ')}`;
+
+  card.appendChild(title);
+  card.appendChild(image);
+  card.appendChild(genres);
+  card.appendChild(likeBtn);
+  card.appendChild(likes);
+  card.appendChild(comment);
+
+  comment.addEventListener('click', () => {
+    const body = document.querySelector('body');
+    window.scrollTo(0, 0);
+    pop.style.display = 'block';
+    body.style.overflow = 'hidden';
+    callPop(movieData);
+  });
+
+  return card;
+};
+
+const createMovieCards = async () => {
+  const response = await fetch(`${apiUrl}`);
+  const showData = await response.json();
+  const shows = showData.slice(0, 20);
+  const count = shows.length;
+  updateMovieCardCount(count);
```

Comment:
Consider not getting the number for the movies by using the length of the array, instead, I recommend querying the class that they have in common and then make you calculations based on that, as you have done for comments counter. This will put your project on the same page with the milestone requirements.